library/common/engine_utility.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "common/stats/utility.h"
+
+#include "library/common/types/c_types.h"
+#include "library/common/types/c_types_utility.h"
+
+namespace Envoy {
+namespace Utility {
+
+Stats::StatNameTagVector transform(envoy_stats_tags tags, Stats::StatNameSetPtr& stat_name_set);
```

Comment:
lets create library/common/stats/utility.h and put this functionality there. under

```c++ 
namespace Envoy {
namespace Stats {
namespace Utility {
```